docs/Model_Assumptions.rst

```diff
@@ -176,12 +176,117 @@ The confersion factors are derived from their `Gasoline Gallon Equivalents.<http
+
+.. list-table:: Limitations
+   :widths: 25 25 25
+   :header-rows: 1
+
+   * - Type
+     - Limitation
+     - Link
+   * - Inherited
+     - Disregard of some real life constraint
+     - :ref:`disregard`
+   * - Inherited
+     - Simplified asset model
+     - 
+   * - Inherited
+     - No degradation
+     - 
+   * - Inherited
+     - Perfect foresight
+     - 
+   * - Can be addressed
+     - Extension of KPIs
+     - 
+   * - Can be addressed
+     - Random excess energy distribution
+     - 
+   * - Can be addressed
+     - Renewable energy share defintion
+     - 
+   * - Can be addressed
+     - Energy carrier rating
+     -      
+   * - Can be addressed
+     - Energy shortage or grid interruption
+     - 
+   * - Can be addressed
+     - Generic cost definitions
+     -  
+   * - Can be addressed
+     - Unlinked one technical unit transformers
+     -      
+
+.. _disregard:
+Disregard of some real life constraint
+######################################
+:Limitation: Some real life constraint are not taken into consideration in the MVS.
+:Reason: The MVS is based oemof-solph python library and uses its generic components to set up an energy system, which allow unidirectional flows only. 
+:Implications: This means that the MVS might result in unfeasible dispatch of assets. For instance, the battery might be charged by the rectifier and discharged by the inverter at the same time step t. Another case that could occur is feeding the grid and consuming from it at the same time t. If those constraints were to be implemented, the optimization problem would become a non-linear one.
+
+Simplified asset model
+######################
+:Limitation: The MVS simplifies the modelling of some assets.
+:Reason: The MVS is based oemof-solph python library and uses its generic components to set up an energy system.
+:Implications: Simplifying the implementation of some component specifications can be beneficial for the ease of the model, however, it contributes to the lack of realism and might result in impractical values. For instance, generators and transformers are modelled without their efficiency curve, turbines without ramp rates, and thermal storage is currently implemented like battery storage.
+
+No degradation
+##############
+:Limitation: The MVS does not degrade the generation of a production asset over the lifetime of the project.
+:Reason: The MVS is based oemof-solph python library and uses its generic components to set up an energy system.
+:Implications: This results in an overestimation of the energy generated by the asset, which implies that the calculation of some other results might be unrealistic (e.g., overestimation of feed-in energy).
```

Comment:
How can the user circumvent this? You shared an idea on monday...

"unrealistic" is a tough call, it is better to describe the effect.